paper.tex

```diff
@@ -35,6 +35,29 @@
 \begin{document}
 \maketitle
 
+\abstract{
+  Symbolic based computer aided algebra systems (CAS) play an important role in
+  scientific computing with the earliest packages dating back into the 1970s.
+  Numerous open and proprietary software packages exist that aid users in
+  complex algebra, calculus, and other mathematical concepts that drive and
+  support both symbolic and numerical computing needs.
+  Here we present a symbolic CAS written in the Python programming language
+  that is developed by a large team of scientists, engineers, and software
+  developers in a bazaar open source development model.
+  The accessibility of the codebase and the community model allow SymPy to
+  rapidly respond to the needs of the community of users without the knowledge
+  or consent of the original developers.
+  This paper will give an overview of what Sympy is and what it offers with
+  respect to similar software.
+  We will highlight the benefit of SymPy's community model and show how this
+  meets instantaneous computing needs of our users.
+  To close, we will present why and how SymPy is useful to researchers,
+  educators, and the scientific software industry with examples of it's use in
+  these fields.
+  The supplementary materials will further outline the details of the design
+  decisions and algorithms that make SymPy what it is today.
```

Comment:
Maybe they do need to be, but do we actually have any algorithms that we've invented ourselves? I'm sure there are some, but it seems that we've mostly implemented existing algorithms and we are only entitle to cite them. Unless we've improved them or something.